### DISCUSSION TITLE
deploy a chatbot ui without API Key

### DISCUSSION THREAD

**Discussion by stanleyshek** [vercel/community#2129] [Help]
### Summary

We are trying out a chatbot ui from github.  However, when deploy in vercel, it forced to provide the API key before deployment.  We tried the author's demo and it can let the user provide it's own API key, is that a limitation on Vercel?

the url of chat chatbot ui is:
https://github.com/mckaywrigley/chatbot-ui

Need help and regards

**Comment by amyegan** [comment]
Hi @stanleyshek! This doesn't sound like a Vercel limitation. I was able to use the [Deploy with Vercel](https://vercel.com/new/clone?repository-url=https%3A%2F%2Fgithub.com%2Fmckaywrigley%2Fchatbot-ui) button to deploy my own copy of the UI without providing an API key upfront.

Can you walk me through the steps you tried, and share screenshots or error logs so I can see exactly what problem you're facing?
